spec/models/piece_spec.rb

```diff
@@ -0,0 +1,124 @@
+require 'rails_helper'
+
+RSpec.describe Piece, type: :model do
+  describe 'is_obstructed?' do
```

Comment:
For instance methods. Put a `#` infront of the method name in describe blocks.

example:
 `describe '#obstructed?'`

mo' info: http://betterspecs.org/#describe